dataland-frontend/tests/e2e/specs/user-authentication/LandingPageRedirect.ts

```diff
@@ -1,10 +1,12 @@
 import { getBaseUrl } from "@e2e/utils/Cypress";
 
-describe("As a user, I expect to get redirected to the company search page when I am logged in and visit the landing page", () => {
+describe("As a user, I expects not to be automatically redirected to the company search page when I am logged in and visit the landing page", () => {
```

Comment:
This description is a little bit cumbersome. Maybe something like:
"As a user, I expect to find a backToPlatformLink when logged in and visiting the landing page"